temba_client/tests.py

```diff
@@ -72,6 +72,9 @@ def test_format_iso8601(self):
         d = datetime.datetime(2014, 1, 2, 3, 4, 5, 6, UtilsTest.TestTZ())
         self.assertEqual(format_iso8601(d), '2014-01-02T08:04:05.000006')
 
+    def test_format_iso8601_should_return_none_when_no_datetime_given(self):
+        self.assertIs(format_iso8601(None), None)
```

Comment:
I think this test can go in https://github.com/rapidpro/rapidpro-python/pull/38/files#diff-954f63ec63275fe04a656fe893fc66c5R78